src/Amadevus.RecordGenerator.Attributes/Features.cs

```diff
@@ -49,9 +49,9 @@ public enum Features
         /// </summary>
         Default = Constructor | Withers | ToString | Builder | Deconstruct,
 
-        //ObjectEquals = 0b_100000,
-        //EquatableEquals = 0b_1000000,
-        //OperatorEquals = 0b_10000000,
-        //Equality = ObjectEquals | EquatableEquals | OperatorEquals
+        ObjectEquals = 0b_100000,
+        EquatableEquals = 0b_1000000,
+        OperatorEquals = 0b_10000000,
+        Equality = ObjectEquals | EquatableEquals | OperatorEquals
```

Comment:
Just a note to add xml-doc comments. Also, I think with the longer numbers, we can group into 4-digit groups separated by `_`. When you're at it, could you fix Deconstruct as well? :) Thanks!

Reply:
👍